model/shared/src/main/scala/explore/modes/ItcInstrumentConfig.scala

```diff
@@ -24,6 +24,7 @@ sealed trait ItcInstrumentConfig derives Eq:
   val grating: Grating
   def gratingDisplay: Display[Grating]
   def gratingStr: String = gratingDisplay.shortName(grating)
```

Comment:
I was tempted to make a deeper structure with ImagingIItcINstrumentConfig and SpectroscopyItcINstrurmentConfig and remove `gratingStr` but it seemed overkill